test/com/tfl/billing/JourneyCostCalculatorTest.java

```diff
@@ -16,56 +18,105 @@
+        }
+        assertTrue(flag);
     }
 
     @Test
-    public void getJourneysTest() throws Exception {
+    public void getTotalTest() {
+        BigDecimal expected = BigDecimal.valueOf(2.4);
+        assertThat(costCalcuator.roundToNearestPenny(costCalcuator.getTotal(customerJourneys, BigDecimal.ZERO)), is(costCalcuator.roundToNearestPenny(expected)));
     }
 
     @Test
-    public void getTotalTest() throws Exception {
+    public void getTotalForCustomerCurrentlyTravelling() {
+        eventLog = Arrays.asList(new JourneyStart(c.cardId(), OysterReaderLocator.atStation(Station.PADDINGTON).id()), new JourneyStart(c1.cardId(), OysterReaderLocator.atStation(Station.HOLBORN).id()), new JourneyEnd(c.cardId(), OysterReaderLocator.atStation(Station.VICTORIA_STATION).id()), new JourneyEnd(c1.cardId(), OysterReaderLocator.atStation(Station.CHANCERY_LANE).id()), new JourneyStart(c.cardId(), OysterReaderLocator.atStation(Station.OXFORD_CIRCUS).id()), new JourneyEnd(c.cardId(), OysterReaderLocator.atStation(Station.VICTORIA_STATION).id()));
+        BigDecimal expected = BigDecimal.valueOf(4.8);
+        assertThat(costCalcuator.roundToNearestPenny(costCalcuator.getTotalForCustomer(c, eventLog)),is(costCalcuator.roundToNearestPenny(expected)));
     }
 
     @Test
-    public void peakTest() {
-        peakTime.setTime(getTimeInMillis(6));
-        offPeakTime.setTime(getTimeInMillis(12));
+    public void getTotalForNotTravellingCustomer() {
+        eventLog = Arrays.asList(new JourneyStart(c.cardId(), OysterReaderLocator.atStation(Station.PADDINGTON).id()), new JourneyStart(c1.cardId(), OysterReaderLocator.atStation(Station.HOLBORN).id()), new JourneyEnd(c.cardId(), OysterReaderLocator.atStation(Station.VICTORIA_STATION).id()), new JourneyEnd(c1.cardId(), OysterReaderLocator.atStation(Station.CHANCERY_LANE).id()), new JourneyStart(c.cardId(), OysterReaderLocator.atStation(Station.OXFORD_CIRCUS).id()), new JourneyEnd(c.cardId(), OysterReaderLocator.atStation(Station.VICTORIA_STATION).id()));
+        BigDecimal expected = BigDecimal.ZERO;
+        assertThat(costCalcuator.roundToNearestPenny(costCalcuator.getTotalForCustomer(c2, eventLog)),is(costCalcuator.roundToNearestPenny(expected)));
+    }
 
-        assertTrue(testCalc.peak(peakTime));
-        assertFalse(testCalc.peak(offPeakTime));
+    @Test
+    public void peakTimeIsMarkedAsPeak() {
+        peakTime.setTime(hoursToMillis(6));
+        offPeakTime.setTime(hoursToMillis(12));
 
-        peakTime.setTime(getTimeInMillis(18));
-        offPeakTime.setTime(getTimeInMillis(22));
+        assertTrue(costCalcuator.peak(peakTime));
+        assertFalse(costCalcuator.peak(offPeakTime));
 
-        assertTrue(testCalc.peak(peakTime));
-        assertFalse(testCalc.peak(offPeakTime));
+        peakTime.setTime(hoursToMillis(18));
+        offPeakTime.setTime(hoursToMillis(22));
 
+        assertTrue(costCalcuator.peak(peakTime));
+        assertFalse(costCalcuator.peak(offPeakTime));
```

Comment:
having multiple assserts in a test is rather a bad practice. https://softwareengineering.stackexchange.com/questions/7823/is-it-ok-to-have-multiple-asserts-in-a-single-unit-test